wx/lib/agw/customtreectrl.py

```diff
@@ -7733,8 +7733,9 @@ def Edit(self, item):
         if self._editCtrl != None and item != self._editCtrl.item():
             self._editCtrl.StopEditing()
 
-        self._editCtrl = TreeTextCtrl(self, item=item)
-        self._editCtrl.SetFocus()
+        if self._editCtrl == None:
```

Comment:
using `is` in place of `==` would probably be better

Reply:
That's how None comparisons are handled everywhere else in file.

Reply:
Done.